frontend/src/pages/pipelines/global/modelCustomization/FineTunePageFooter.tsx

```diff
@@ -33,8 +45,105 @@ const FineTunePageFooter: React.FC<FineTunePageFooterProps> = ({ isInvalid, onSu
   // TODO: translate data to `RunFormData`
   const [runFormData] = useRunFormData(null, {});
 
+  const onSubmit = async (dryRun: boolean) => {
+    const [teacherSecret, judgeSecret] = await createTeacherJudgeSecrets(
+      namespace,
+      data.teacher,
+      data.judge,
+      dryRun,
+    );
```

Comment:
In this way, the dry-run creation and the real creation will generate 2 different secret names. IMO it's fine because as long as we are setting the real created secret names to the parameters, it's OK. The dry run will only test whether there are any other problems or not.
However, if anyone who reviews the PR thinks that we need to use the same secret names for both dry-run creation and real creation, please let me know, I will think of another way to store the dry-run created names and reuse the names when we really create the secrets.